package.json

```diff
@@ -1,5 +1,32 @@
 {
+  "name": "istanbuljs",
+  "version": "1.0.0",
```

Comment:
We're calling this the supporting libraries for the IstanbulJS 2.0 CLI, so shouldn't this start at 2.0.0?